services/traction/api/resources/tenant.py

```diff
@@ -15,13 +17,28 @@
 router = APIRouter()
 
 
+class TenantWallet(BaseModel):
+    created_at: str
+    key_management_mode: str
+    settings: dict
+    state: Optional[str] = None
```

Comment:
python 3.10 lets you just go `state: str | None = None` instead of needing the `typing.Optional`. 

https://www.python.org/dev/peps/pep-0604/